doc/examples/species/1.json

```diff
@@ -1,5 +1,6 @@
 {
   "id": 1,
   "name": "Baobab Tree",
+  "scientificName": "Adansonia",
```

Comment:
@VWRoli you must remove this field, because that we can not provide this now, from the database.

Reply:
Removed it.
![Screenshot from 2021-12-02 08-19-22](https://user-images.githubusercontent.com/31432331/144376396-b955924f-5691-4e1b-9ce6-570528124f87.png)